File: Homework3/Question2/src/Bird.java
public class Bird extends Animal {
    private int numberOfFeathers;

    public Bird(String color, int numberOfFeathers) {
        super(color);
        this.numberOfFeathers = numberOfFeathers;
    }

    public int getNumberOfFeathers() {
        return numberOfFeathers;
    }

    @Override
    public boolean isEdible() {
        return true;
    }

    public static void testBird() {
        Bird bird = new Bird("orange", 8700);
        //Test attributes of bird
        System.out.println(bird.getNumberOfFeathers() == 8700);
        System.out.println(bird.isEdible() == true);
        System.out.println(bird.getColor() == "orange");
    }
}

File: Homework3/Bird.kt
open class Bird : Animal {
    private val numberOfFeathers: Int

    constructor(color: String, numberOfFeathers: Int) : super(color) {
        this.numberOfFeathers = numberOfFeathers
    }

    fun getNumberOfFeathers(): Int {
        return numberOfFeathers
    }

    override fun isEdible(): Boolean {
        return true
    }

    companion object {
        fun testBird() {
            val bird = Bird("orange", 8700)
            println(bird.getNumberOfFeathers() == 8700)
            println(bird.isEdible() == true)
            println(bird.getColor() == "orange")
        }
    }
}